tests/conftest.py

```diff
@@ -17,7 +17,7 @@
 from typing import Union
 
 import pytest
-from psycopg2.pool import ThreadedConnectionPool
+from psycopg_pool import ConnectionPool as PgConnectionPool
```

Comment:
Could you move this import to the top of `create_postgres_bucket`? It's not used elsewhere, and that would make it possible for us to deselect the tests without forcing users to install `psycopg_pool`.